### DISCUSSION TITLE
Appointment reptetition on certains days

### DISCUSSION THREAD

**Discussion by Zenkenkongo** [tutao/tutanota#5804] [Ideas]
There are already reptetition of appointment of intervalls.

I would like the following extension:

- appointment reptetition on a specific day of the week of the month, f.e. second Tuesday of the month is meeting --> 12.09.2023, 10.10.2023, 14.11.2023, ...

**Comment by vitoreiji** [comment]
That is already planned =) https://github.com/tutao/tutanota/issues/1666
